library/VTKExtensions/Rendering/vtkF3DDropZoneActor.cxx

```diff
@@ -0,0 +1,167 @@
+#include "vtkF3DDropZoneActor.h"
+
+#include <vtkObjectFactory.h>
+#include <vtkPolyData.h>
+#include <vtkPolyDataMapper2D.h>
+#include <vtkTextMapper.h>
+#include <vtkTextProperty.h>
+#include <vtkViewport.h>
+
+constexpr int tickWidth = 3;
+constexpr int tickLength = 10;
+
+vtkStandardNewMacro(vtkF3DDropZoneActor);
+
+//------------------------------------------------------------------------------
+vtkF3DDropZoneActor::vtkF3DDropZoneActor()
+{
+  this->TextActor->SetMapper(this->TextMapper);
+  this->TextMapper->SetInput("Drop a file to open it\nPress H to show cheatsheet");
+  vtkTextProperty* tProp = this->TextMapper->GetTextProperty();
+  tProp->SetFontSize(25);
+  tProp->SetJustificationToCentered();
+  tProp->SetVerticalJustificationToCentered();
+
+  this->BorderMapper->SetInputData(this->BorderPolyData);
+  this->BorderActor->SetMapper(this->BorderMapper);
+}
+
+//------------------------------------------------------------------------------
+vtkF3DDropZoneActor::~vtkF3DDropZoneActor() = default;
+
+//------------------------------------------------------------------------------
+void vtkF3DDropZoneActor::ReleaseGraphicsResources(vtkWindow* win)
+{
+  this->TextActor->ReleaseGraphicsResources(win);
+  this->BorderActor->ReleaseGraphicsResources(win);
+}
+
+//------------------------------------------------------------------------------
+int vtkF3DDropZoneActor::RenderOverlay(vtkViewport* viewport)
+{
+  const int* vSize = viewport->GetSize();
+
+  // Simply position text actor in the middle
+  this->TextActor->SetPosition(vSize[0] / 2, vSize[1] / 2);
```

Comment:
the position here is the center of the actor?

Reply:
yes, since the text actor is justified in both directions.